RasiakMykola/HW3/Task_2 .py

```diff
@@ -0,0 +1,23 @@
+
+# 03.1 PRACTICAL TASKS / [GITHUB]
+
+
+# Task 2
+four_digit_natural_number = "6274"
+
+a, b, c, d = four_digit_natural_number[0], four_digit_natural_number[1], \
+             four_digit_natural_number[2], four_digit_natural_number[3]
+
+the_product_of_the_digits = int(a)*int(b)*int(c)*int(d)
+
```

Comment:
more optimal version of code 
number = input("Enter a four digit number:")

print(f'The product of numbers is: {int(number[0]) * int(number[1]) * int(number[2]) * int(number[3])}')

print(f'Reversed number: {number[::-1]}')

print(f'Sorted list: {list("".join(sorted(number)))}\n')